src/neo-vm/Types/Null.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Neo.VM.Types
+{
+    internal class Null : StackItem
+    {
+        public override bool Equals(StackItem other)
+        {
+            if (ReferenceEquals(this, other)) return true;
+            if (other is null) return true;
+            if (other is Null) return true;
+            return false;
+        }
+
+        public override bool GetBoolean()
+        {
+            return false;
+        }
+
+        public override byte[] GetByteArray()
+        {
+            throw new NotSupportedException();
```

Comment:
`NUMEQUAL` will fail, this is the expected behaviour?

Reply:
`NUMEQUAL` is for numbers.